src/main/java/io/Out.java

```diff
@@ -0,0 +1,153 @@
+package io;
+
+import tasks.Store;
+import tasks.Task;
+
+public class Out {
+    private static final int BOX_WIDTH = 80;
+
+    private static void printLineWithDelim(char leftDelim, char rightDelim) {
+        System.out.print(leftDelim);
+        for (int i = 0; i < BOX_WIDTH - 2; i++) {
+            System.out.print(BoxDrawingCharacter.HORIZONTAL_LINE);
+        }
+        System.out.println(rightDelim);
+    }
+
+    private static void printBoxTopBorder() {
+        printLineWithDelim(BoxDrawingCharacter.TOP_LEFT_CORNER, BoxDrawingCharacter.TOP_RIGHT_CORNER);
+    }
+
+    private static void printBoxBottomBorder() {
+        printLineWithDelim(BoxDrawingCharacter.BOTTOM_LEFT_CORNER, BoxDrawingCharacter.BOTTOM_RIGHT_CORNER);
+        System.out.println("");
+    }
+
+    private static void printBoxBottomBorder(boolean withBottomMargin) {
+        printLineWithDelim(BoxDrawingCharacter.BOTTOM_LEFT_CORNER, BoxDrawingCharacter.BOTTOM_RIGHT_CORNER);
+        if (withBottomMargin) {
+            System.out.println("");
+        }
+    }
+
+    private static void printBoxRightBorder(int unusedSpace) {
+        // '<=' not '<' since we also have one space as padding
+        for (int i = 0; i <= unusedSpace; i++) {
+            System.out.print(' ');
+        }
+        System.out.println(BoxDrawingCharacter.VERTICAL_LINE);
+    }
+
+    /**
+     * Print the logo with padding and all.
+     * 
+     * @see {@link https://patorjk.com/software/taag/#p=display&f=Slant&t=archduke}
+     */
+    private static void printLogo() {
+        String left = BoxDrawingCharacter.VERTICAL_LINE + " ";
+        String right = " " + BoxDrawingCharacter.VERTICAL_LINE + "\n";
+        String[] lines = { "                   __        __      __      ",
+                "  ____ ___________/ /_  ____/ /_  __/ /_____ ", " / __ `/ ___/ ___/ __ \\/ __  / / / / //_/ _ \\",
+                "/ /_/ / /  / /__/ / / / /_/ / /_/ / ,< /  __/", "\\__,_/_/   \\___/_/ /_/\\__,_/\\__,_/_/|_|\\___/ ",
+                "                                             " };
```

Comment:
Line 50 and line 51 can be split into 4 separate lines, computers with smaller aspect ratios cannot see line 51 on the same line.

Reply:
Actually it's due to my formatter, the line length limit is set to 120 chars as per [recommendation](https://se-education.org/guides/conventions/java/basic.html#layout) and the editor took charge of automatically add new lines where needed. I think I will reduce the length limit to 100 chars instead.

Reply:
Line length limit reduced to 100 chars.